core/server/src/main/java/alluxio/worker/file/FileDataManager.java

```diff
@@ -69,6 +72,15 @@
   private final Configuration mConfiguration;
   private final Object mLock = new Object();
 
+  /** Provides a per thread rate limiter to throttle async persistence. */
+  private static ThreadLocal<RateLimiter> sPersistenceRateLimiter = new ThreadLocal<RateLimiter>() {
```

Comment:
Oh I misunderstood what we talked last time, yeah, this should be changed to one RateLimiter instead of a thread-local. Still, this is gonna be a per worker rate limiter, does that sound okay to you?